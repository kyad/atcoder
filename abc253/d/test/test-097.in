10000000 9800 5032
