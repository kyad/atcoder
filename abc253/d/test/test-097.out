49984969613704
